src/main/resources/static/openapi.yaml

```diff
@@ -428,106 +162,13 @@ components:
         passwordAgain:
           format: password
           type: string
+        city:
```

Comment:
this should be part of settings request, not from the the user registration/password change